docs/Reference/EthSigner-CLI.md

```diff
@@ -105,31 +156,66 @@ File containing password for the [key with which transactions are signed](../Usi
 --password-file=/Users/me/my_node/password
 ```
 
-### logging
+## Hashicorp Options 
 
-Logging verbosity levels. Options are: `OFF`, `FATAL`, `WARN`, `INFO`, `DEBUG`, `TRACE`, `ALL`. The 
-default is `INFO`.  
+### auth-file
+
+File containing authentication data for Hashicorp Vault. The authentication data is the [root token displayed by
+the Hashicorp Vault server](../Using-EthSigner/Hashicorp.md#storing-private-key-in-hashcorp-vault). 
 
 ```bash tab="Syntax"
--l, --logging=<LOG VERBOSITY LEVEL>
+--auth-file=<authFile>
 ```
 
 ```bash tab="Example"
---logging=DEBUG
+--auth-file=/Users/me/my_node/auth_file
 ```
 
-### help
+### host
 
-Displays the help and exits.  
+Host of the Hashicorp Vault server. Default is `localhost`. 
 
 ```bash tab="Syntax"
--h, --help
+--host=<serverHost>
 ```
 
-### version
+```bash tab="Example"
+--host="http://host.com"
+```
 
-Displays the version and exits.  
+### port
+
+Port of the Hashicorp Vault server. Default is 8200. 
 
 ```bash tab="Syntax"
--V, --version
-```
\ No newline at end of file
+--port=<serverPort>
+```
+
+```bash tab="Example"
+--port=23000
+```
+
+### signing-key-path
+
+Path to secret in the Hashicorp Vault containing the private key for signing transactions. Default is
+` /secret/data/ethsignerSigningKey`. 
+
+```bash tab="Syntax"
+--signing-key-path=<signingKeyPath>
+```
+
+```bash tab="Example"
+--signing-key-path=/my_secret/ethsignerSigningKey
+```
+
+### timeout
+
+Timeout in seconds for requests to the Hashicorp Vault server. Default is 10. 
```

Comment:
probably going to be changed to milliseconds?

Reply:
This has now been changed to milliseconds

Reply:
Fixed